packages/zql/src/ivm/exists.ts

```diff
@@ -97,125 +119,125 @@ export class Exists implements Operator {
   }
 
   push(change: Change) {
-    switch (change.type) {
-      // add, remove and edit cannot change the size of the
-      // this.#relationshipName relationship, so simply #pushWithFilter
-      case 'add':
-      case 'edit': {
-        this.#pushWithFilter(change);
-        return;
-      }
-      case 'remove': {
-        const size = this.#getSize(change.node);
-        // If size is undefined, this operator has not output
-        // this row before and so it is unnecessary to output a remove for
-        // it.  Which is fortunate, since #fetchSize would
-        // not be able to fetch a Node for this change since it is
-        // removed from the source.
-        if (size === undefined) {
+    this.#inPush = true;
+    try {
```

Comment:
git is doing a poor job with the diff here, very little has change here, it is just wrapped in a try / finally.